app/urls.py

```diff
@@ -57,5 +58,15 @@
         'users/<uuid:pk>/',
         UserDetail.as_view(),
         name='user_detail'
-    )
+    ),
+    path(
+        'interests/',
+        InterestList.as_view(),
+        name='interest_list'
+    ),
+    path(
+        'interests/<int:pk>/',
```

Comment:
https://github.com/mentorpaired/core/issues/41#issue-598318640 same suggestion as the above one.